pkg/api/rpc.go

```diff
@@ -43,6 +47,11 @@ type BuildPurgeRequest struct {
 	Testplan string `json:"testplan"`
 }
 
+type TaskStatusRequest struct {
+	ID                string `json:"id"`
+	WaitForCompletion bool   `json:"wait-for-completion"`
```

Comment:
Seems like you are using both camelCase and other naming scheme - I suggest we `snake_case` everywhere - I think this is what Testground is already using (i.e. "run_config", "build_config", "run_id", etc.)